### PR TITLE
Update links from code.google.com to github

### PR DESCRIPTION
**Pull request by mulinnuha**
Motivation:

This pull request does not solve any problem but we find that several links in the code refer to project websites under the domain of http://code.google.com which are either moved to github or not maintained anymore.

Modification:

Update the project links from code.google.com to the relevant project in github.com

### CODE REVIEW

File: codec/src/main/java/io/netty/handler/codec/compression/SnappyFrameDecoder.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 /**
  * Uncompresses a {@link ByteBuf} encoded with the Snappy framing format.
  *
- * See http://code.google.com/p/snappy/source/browse/trunk/framing_format.txt
+ * See https://github.com/google/snappy/blob/master/framing_format.txt
```
Comment (Scottmitch) [inline codec/src/main/java/io/netty/handler/codec/compression/SnappyFrameDecoder.java]: can you make this a proper hyperlink (`<a href=".."></a>`) too please?

File: codec/src/main/java/io/netty/handler/codec/compression/SnappyFrameEncoder.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
 /**
  * Compresses a {@link ByteBuf} using the Snappy framing format.
  *
- * See http://code.google.com/p/snappy/source/browse/trunk/framing_format.txt
+ * See https://github.com/google/snappy/blob/master/framing_format.txt
```
Comment (Scottmitch) [inline codec/src/main/java/io/netty/handler/codec/compression/SnappyFrameEncoder.java]: can you make this a proper hyperlink (`<a href=".."></a>`) too please?

**Comment by mulinnuha** [comment]
@Scottmitch updated with proper hyperlinks. thanks!

**Comment by normanmaurer** [comment]
@mulinnuha can you please squash your commits ? Thanks a lot!

**Comment by Scottmitch** [comment]
@mulinnuha - can you sign the ICLA [1] and then I will merge.

[1] https://docs.google.com/forms/d/e/1FAIpQLSd7Bzje39G__THDJLRgKpQZ4gODNE26x_hZW3ofQOkgL6RGCA/viewform?formkey=dHBjc1YzdWhsZERUQnhlSklsbG1KT1E6MQ

**Comment by mulinnuha** [comment]
@normanmaurer @Scottmitch I've squashed the commits and signed the ICLA. Thank you :)

**Comment by normanmaurer** [comment]
Cherry-picked into 4.1 (a9d7c72cde9d522161e7275df6490a8b9765d57f) and 4.0 (d1865406edb0e81e400f69c75c50f638275ce6f9)

Thanks a lot....

**Pull request closed without merging**